README.md

```diff
@@ -1,61 +1,65 @@
 # Megalodon
+
 [![Test](https://github.com/h3poteto/megalodon/workflows/Test/badge.svg)](https://github.com/h3poteto/megalodon/actions?query=branch%3Amaster+workflow%3ATest)
 [![NPM Version](https://img.shields.io/npm/v/megalodon.svg)](https://www.npmjs.com/package/megalodon)
 [![GitHub release](https://img.shields.io/github/release/h3poteto/megalodon.svg)](https://github.com/h3poteto/megalodon/releases)
 [![npm](https://img.shields.io/npm/dm/megalodon)](https://www.npmjs.com/package/megalodon)
 [![NPM](https://img.shields.io/npm/l/megalodon)](/LICENSE.txt)
 
-A Fediverse API Client library for node.js and browser. It provides REST API and streaming methods.
-By using this library, you can take Mastodon, Pleroma, Friendica, and Firefish with the same interface.
+Megalodon is a Fediverse API client library for [NodeJS](https://nodejs.org) and browsers.
+This library allows for interfacing with [Mastodon](https://joinmastodon.org), [Pleroma](https://pleroma.social), [Friendica](https://friendi.ca), and [Firefish](https://joinfirefish.org) servers all with the same interface, providing REST API and streaming methods.
 
 The Rust version is [megalodon-rs](https://github.com/h3poteto/megalodon-rs).
 
-## Supporting
-- [x] Mastodon
-- [x] Pleroma
+## Supports
+
+- [x] Mastodon <img src="https://cdn.simpleicons.org/mastodon" alt="Mastodon" width=16 height=16>
```

Comment:
@ThatOneCalculator Can you please check this link?
I can't see the image with this URL.

Reply:
Yeah, for some reason it seems to have corrupted when turning into a camo link. It worked fine in the Github preview

Reply:
Or not...? I'll change the images anyways since it seems to be acting strangely

![image](https://github.com/h3poteto/megalodon/assets/44733677/b09e9662-a05d-441b-8ea9-d3cb6597eefc)

Reply:
Github's CDN seems to have been just caching the images strangely for a little bit. Works on desktop, mobile, etc now

Reply:
Oh, it's working now.